tests/test_app.py

```diff
@@ -1253,6 +1255,52 @@ async def test_bulk_specification_dts_fail_wrong_extension(manifest: str, expect
             assert resp.status == 400
 
 
+def test_bulk_specification_dts_fail_bad_schema():
+    # TODO: This is tested manually, as there's no good way to inject bad configs
+    # to individual tests right now.
+    # TODO: automated tests for:
+    # * missing schema config
+    # * missing schema file
+    # * malformed schema file (i.e. not json)
+    # * bad schema (good JSON, invalid as json schema)
+    pass
+
+
+def test_load_and_validate_schema_good():
+    # TODO: update this after updating how config is handled
+    schema_file = config["staging_service"]["DTS_MANIFEST_SCHEMA"]
+    validator = app.load_and_validate_schema(schema_file)
+    assert isinstance(validator, jsonschema.Draft202012Validator)
+
+
+def test_load_and_validate_schema_missing_file():
+    not_real_file = Path("not_real")
+    while not_real_file.exists():
+        not_real_file = Path(str(uuid.uuid4()))
+    with pytest.raises(FileNotFoundError, match="No such file or directory"):
+        app.load_and_validate_schema(not_real_file)
+
+
+def test_load_and_validate_schema_malformed_file(tmp_path: Path):
+    # TODO: migrate FileUtil and import_specifications.test_individual_parsers.temp_path_fixture
+    # into conftest.py, and resolve everywhere else that FileUtil gets used.
+    # Until then, the built-in tmp_path is appropriate for these tests
+    wrong_schema = "not valid json"
+    schema_file = tmp_path / f"{uuid.uuid4()}.json"
+    schema_file.write_text(wrong_schema, encoding="utf-8")
+    with pytest.raises(json.JSONDecodeError, match="Expecting value: line 1 column 1"):
```

Comment:
This seems like the wrong error, naively - there's a value there, it's just the wrong value

Reply:
That's the error provided by the json decoder. It's expecting a JSON value, not a bare string.